src/types/user.types.ts

```diff
@@ -104,7 +104,9 @@ interface CristinPersonName extends CristinArrayValue {
 }
 
 interface CristinPersonContactDetails {
-  telephone: string;
+  telephone?: string | undefined;
+  email?: string | undefined;
+  webPage?: string | undefined;
```

Comment:
`?` er bare syntactic sugar for `undefined`
```suggestion
  telephone?: string;
  email?: string;
  webPage?: string;
```